php/Terminus/Models/Collections/Instruments.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Terminus\Models\Collections;
+
+class Instruments extends TerminusCollection {
+
+  /**
+   * Instantiates the collection, sets param members as properties
+   *
+   * @param [array] $options To be set to $this->key
+   * @return [Instruments] $this
+   */
+  public function __construct($options = array()) {
+    $option_args = array_merge(
+      $options, 
+      array(
+        'fetch_url'  => 'users/' . $options['user']->get('id') . '/instruments',
```

Comment:
Rather than defining this as a constant during construction, I like defining "url()" as a method of the instance. That way all the URL logic is both encapsulated (eg not hidden away in the constructor) and dynamic.